plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -93,11 +93,14 @@
   private DataBindingContext bindingContext;
 
   private Runnable layoutChangedHandler;
+  private boolean childOfPropertyDialog = false;
```

Comment:
Not in this PR, but we need it very soon when we fix #759. (Note that #742 added an account selector to the deploy preference.) We want the account choice to be an optional setting, for the same reason as project ID.